plugin/storage/cassandra/factory.go

```diff
@@ -28,6 +28,7 @@ import (
 	"github.com/jaegertracing/jaeger/storage"
 	"github.com/jaegertracing/jaeger/storage/dependencystore"
 	"github.com/jaegertracing/jaeger/storage/spanstore"
+	"github.com/jaegertracing/jaeger/storage/spanstore/decorator"
```

Comment:
why the "decorator" package? Metrics is also a decorator, it's in its own package. I suggest `spanstore/ratelimited` instead